workers/gameplay/src/bots.rs

```diff
@@ -397,9 +488,12 @@ impl Game {
         ))
     }
 }
-
 impl Drop for Game {
     fn drop(&mut self) {
+        unsafe {
+            let _ = kill(-(self.bot_a.id().unwrap_or_default() as i32), 18);
+            let _ = kill(-(self.bot_b.id().unwrap_or_default() as i32), 18);
+        };
```

Comment:
I thought the `id()` method returns pid, and `kill(-x)` assumes `x` is a gpid

Reply:
Yes but i set the gpid to the id when running `Command::new`. ` .process_group(0)` sets the gpid of the child to it's own pid, and it's children inherits it.

Reply:
I see, that's cool